src/lib/containers/dataaccess/SubmissionPage.tsx

```diff
@@ -29,14 +29,13 @@ import UserCard from '../UserCard'
 import UserOrTeamBadge from '../UserOrTeamBadge'
 import { FileHandleLink } from '../widgets/FileHandleLink'
 import duration from 'dayjs/plugin/duration'
+import RejectDataAccessRequestModal from './RejectDataAccessRequestModal'
 
 dayjs.extend(duration)
 
 export type SubmissionPageProps = {
   /** The ID of the submission to view */
   submissionId: string | number
-  /** Invoked when a reviewer clicks "Reject". Provides a parameter for an external component to provide a reason for rejection, which will reject the submission when invoked. */
-  onRejectClicked: (onReject: (rejectedReason: string) => void) => void
```

Comment:
No longer need this prop since we don't need to show the existing SWC modal. Handle the entire flow in this component.